bin/common/usr_local_bin_to_path.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Add /usr/local/bin to non-login path, since tools are installed there
+if ! grep ENV_SUPATH /etc/login.defs | grep -q /usr/local/bin; then
+  sed -i '/ENV_SUPATH/s/$/:\/usr\/local\/bin/' /etc/login.defs
```

Comment:
Is `/etc/environment.d` insufficient for this? Which non-login path are we talking about here?